diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/BlockStructureBracesFixTest.kt

```diff
@@ -43,4 +43,10 @@ class BlockStructureBracesFixTest : FixTestBase ("test/paragraph3/block_brace",
     fun `should fix open and close brace in try-catch-finally expression`() {
         fixAndCompare("TryBraceExpected.kt", "TryBraceTest.kt")
     }
+
+    @Test
+    @Tag(WarningNames.BRACES_BLOCK_STRUCTURE_ERROR)
+    fun `should fix open and close brace in common expression`() {
+        fixAndCompare("CommonExpected.kt", "CommonTest.kt")
```

Comment:
Was it after all actually failing on this example? I don't remember, what we found out in the end.

If it wasn't failing on this particular rule, please move it to smoke test as an additional example.